src/util/strutil.h

```diff
@@ -0,0 +1,15 @@
+/*
+* Copyright 2018-2020 Redis Labs Ltd. and Contributors
+*
+* This file is available under the Redis Labs Source Available License Agreement
+*/
+
+#pragma once
+
+#include <glob.h>
```

Comment:
What is this include for?
if it is needed in the `c` file please move it there.